tests/views/test_path.py

```diff
@@ -19,7 +19,7 @@ def test_valid_repo_name_for_manifest(self):
 
         self.assertEqual(response.status_code, 302)
         self.assertTrue(response.headers['Content-Type'].startswith('text/html'))
-        self.assertTrue('foo/bar/manifests/2' in response.headers['Location'])
+        self.assertTrue('foo/bar/manifests/1' in response.headers['Location'])
```

Comment:
Is this just a lazy way of asserting that the redirect URL ends in ``foo/bar/manifests/1.25.1-musl`` ?